src/CommonITILObject.php

```diff
@@ -11281,4 +11281,19 @@ public function canRequesterUpdateItem()
     {
         return true;
     }
+
+    /**
+     * Retrieve recipient user ID
+     *
+     * @return int|false
+     */
+    public function getRecipientUserID(): int|false
+    {
+        $users_id = $this->fields['users_id_recipient'] ?? false;
+        $user = new User();
+        if (!$users_id || $user->getFromDB($users_id)) {
```

Comment:
```suggestion
        if (!$users_id || !$user->getFromDB($users_id)) {
```

Reply:
I was that issue adding test. I also changed to handle the `=== 0` case like it was in template